holoclean.py

```diff
@@ -121,6 +121,12 @@
       'default': 0.1,
       'type': float,
       'help': 'Correlation threshold (absolute) when selecting correlated attributes for domain pruning.'}),
+    (('-wlt', '--estimator-enabled'),
+     {'metavar': 'ESTIMATOR_ENABLED',
+      'dest': 'estimator_enabled',
+      'default': True,
+      'type': bool,
+      'help': 'Enables the posterior estimator for weak labelling and domain generation.'}),
```

Comment:
We should add either here or up in the domain model when we skip if this parameter is false (better in my opinion) that we use the top co-occurring values as the cell domain (if that is a valid statement)

Reply:
I'm going to do a large refactor of naming of the parameters and what's surfaced after I finalize the experiments. Going to merge this into `dev` for consistency.